requirements.txt

```diff
@@ -27,11 +30,17 @@ psycopg2-binary==2.9.3
 pydantic<1.10.0
 pydash==5.1.1
 PyJWT==2.4.0
+pymongo==3.13.0
+PyMySQL==1.0.2
 python-jose[cryptography]==3.3.0
 pyyaml>=5,<6
+redis==3.5.3
+sqlalchemy[asyncio]==1.4.27
```

Comment:
Is the version change here required?

Reply:
@PSalant726 I think I was getting some small version issues since it is now in the same file as `sqlalchemy` base, and the small bump fixed it